{{cookiecutter.project_slug}}/pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = {{ cookiecutter.project_slug }}.test_settings
+python_files = tests.py test_*.py *_tests.py
+addopts = -p no:warnings --strict-markers --no-migrations --reuse-db
+mccabe-complexity=5
```

Comment:
Alternatively we could change this to 8 or 9 to silence mccabe 😬

Reply:
McCabe's "official" recommendation was to set the threshold to 10 and NIST says that in some cases it can be relaxed to 15 ([citation](https://en.wikipedia.org/wiki/Cyclomatic_complexity#cite_note-mccabe76-3))

I'm in favor of taking that recommendation and setting this to 10 - it would be great if we could be warned about functions in the 5-10 range. maybe we can use SonarCloud or something for that.

Reply:
Can you point me at the complex functions in the utils folder? It's likely I wrote those and wouldn't mind thinking about how to simplify them.

Reply:
for some reason I thought 5 was desired. But OK, 10 it will be for now.
I deleted the test codebase from my local so I don't have the report, but I believe it was:
`_validate_csv()`
`send_html_email()`
and there was a 3rd one, I think probably `_load_csv()`

Reply:
for some reason I thought 5 was desired. But OK, 10 it will be for now.
I deleted the test codebase from my local so I don't have the report, but I believe it was:
`_validate_csv()`
`send_html_email()`
and there was a 3rd one, I think probably `_load_csv()`

Reply:
mccabe change made